third-culture-software/bhima
# Fixed sorting bug for numerical columns in Staffing Indices Management page

Human Resources > Indices > Staffing indices management

Changed column type to 'number' so that sorting by those columns works correctly.

Closes https://github.com/IMA-WorldHealth/bhima/issues/6216

**TESTING**
- Try the page above
- Try sorting by the "Enrollment bonus" and "Function bonus" columns and verify that the sorting is correct now.
